thothTemplate required for thoth advise

harshad16 opened the pull request:
thothTemplate required for thoth advise
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

## This introduces a breaking change

- [x] Yes
- [ ] No

The build with thoth-advise fails as it looks for .thothTemplate.
## This Pull Request implements

Introduced the .thothTemplate.yaml

## Description

standard template file required for thoth advise :smile:

fridex commented on .thothTemplate.yaml:
```diff
@@ -0,0 +1,14 @@
+runtime_environments:
+  - name: '{os_name}:{os_version}'
+    operating_system:
+      name: 'ubi'
```
```
name: 'rhel'
version: '8.0'
```

Maybe we need to change this also in other repos. As we use UBI:8, we should be able to directly use the autodiscovery parameters:

```
name: '{os_name}'
version: '{os_version}'
```

harshad16 replied:
fair point :+1:

harshad16 commented:
Modified .thoth.yaml to use os_name and os_version to be `rhel:8.0`; and switched to .thoth.yaml as config file for builds. we can ease switch once we have rhel:8.1 support to .thothTemplate.yaml.

harshad16 commented:
@fridex  PTAL
